### PR TITLE
SearchOverlay component

### PR DESCRIPTION
**Pull request by sahilmob**

### CODE REVIEW

File: kibbeh/src/ui/SearchOverlay.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+export interface SearchOverlayProps {
+  children: ReactElement;
+}
+
+export const SearchOverlay: React.FC<SearchOverlayProps> = ({ children }) => {
+  return (
+    <div
+      style={{ borderRadius: "8px" }}
+      className="w-full p-2 bg-primary-800 border border-primary-700"
```
Comment (ajmnz) [inline kibbeh/src/ui/SearchOverlay.tsx]: Let's leave this set up for when results come in and add `pb-5`.

File: kibbeh/src/stories/SearchOverlay.story.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import { SearchOverlay, SearchOverlayProps } from "../ui/SearchOverlay";
+import { SearchBar } from "../ui/SearchBar";
+
+export default {
+  title: "SearchOverlay",
+  component: SearchOverlay,
+};
+
+export const Main: Story<SearchOverlayProps> = ({ children }) => (
+  <SearchOverlay>{children || <SearchBar />}</SearchOverlay>
```
Comment (ajmnz) [inline kibbeh/src/stories/SearchOverlay.story.tsx:13]: `SearchBar` will eventually need `pb-2`, but we'll set that up when we start putting everything together.

**Review by ajmnz — CHANGES_REQUESTED** [review]

File: kibbeh/src/ui/SearchOverlay.tsx (hunk 1 above)
Comment (sahilmob) [reply kibbeh/src/ui/SearchOverlay.tsx]: Do you mean that I have to completely remove the padding for now?

Comment (ajmnz) [reply kibbeh/src/ui/SearchOverlay.tsx]: No, just remove `p-2` and add `px-2 pt-2 pb-5`.

**Pull request merged**